test/suites/integration/scheduling_test.go

```diff
@@ -2,20 +2,142 @@ package integration_test
+					g.Expect(env.Client.List(env, podList, client.MatchingLabels(deployment.Spec.Selector.MatchLabels))).To(Succeed(), failureDescription)
+					g.Expect(env.Client.Get(env, client.ObjectKeyFromObject(deployment), deployment)).To(Succeed(), failureDescription)
+					g.Expect(deployment.Status.ReadyReplicas).To(Equal(*deployment.Spec.Replicas), failureDescription)
+				}).Should(Succeed())
+
+				bindings := map[*v1.Pod]*v1.Node{}
+				for _, pod := range podList.Items {
+					node := ExpectScheduled(env.Context, env.Client, &pod)
+					bindings[&pod] = node
+				}
+				expectations(bindings)
+			}(t)
+		}
+		wg.Wait()
+	})
+})
+
+func WellKnownLabelsTest() (Setup, Expectation) {
+	return func(provisioner *v1alpha5.Provisioner, nodeTemplate *v1alpha1.AWSNodeTemplate) *appsv1.Deployment {
+			nodeSelector := map[string]string{
+				// Well Known
+				v1.LabelTopologyRegion:     env.Region,
+				v1.LabelTopologyZone:       fmt.Sprintf("%sa", env.Region),
+				v1.LabelInstanceTypeStable: "g4dn.8xlarge",
+				v1.LabelOSStable:           "linux",
+				v1.LabelArchStable:         "amd64",
+				v1alpha5.LabelCapacityType: "on-demand",
+				// Well Known to AWS
+				awsv1alpha1.LabelInstanceHypervisor:      "nitro",
+				awsv1alpha1.LabelInstanceCategory:        "g",
+				awsv1alpha1.LabelInstanceGeneration:      "4",
+				awsv1alpha1.LabelInstanceFamily:          "g4dn",
+				awsv1alpha1.LabelInstanceSize:            "8xlarge",
+				awsv1alpha1.LabelInstanceCPU:             "32",
+				awsv1alpha1.LabelInstanceMemory:          "131072",
+				awsv1alpha1.LabelInstancePods:            "58", // May vary w/ environment
+				awsv1alpha1.LabelInstanceGPUName:         "t4",
+				awsv1alpha1.LabelInstanceGPUManufacturer: "nvidia",
+				awsv1alpha1.LabelInstanceGPUCount:        "1",
+				awsv1alpha1.LabelInstanceGPUMemory:       "16384",
+				awsv1alpha1.LabelInstanceLocalNVME:       "900",
+				// Deprecated Labels
+				v1.LabelFailureDomainBetaZone:   fmt.Sprintf("%sa", env.Region),
+				v1.LabelFailureDomainBetaRegion: env.Region,
+				"beta.kubernetes.io/arch":       "amd64",
+				"beta.kubernetes.io/os":         "linux",
+				v1.LabelInstanceType:            "g4dn.8xlarge",
+			}
+			Expect(lo.Keys(nodeSelector)).To(ContainElements(append(v1alpha5.WellKnownLabels.UnsortedList(), lo.Keys(v1alpha5.NormalizedLabels)...)))
+			requirements := lo.MapToSlice(nodeSelector, func(key string, value string) v1.NodeSelectorRequirement {
+				return v1.NodeSelectorRequirement{Key: key, Operator: v1.NodeSelectorOpIn, Values: []string{value}}
+			})
+			return test.Deployment(test.DeploymentOptions{Replicas: 1, PodOptions: test.PodOptions{
+				NodeSelector:     nodeSelector,
+				NodePreferences:  requirements,
+				NodeRequirements: requirements,
+			}})
+		},
+		func(bindings map[*v1.Pod]*v1.Node) {
+			for pod, node := range bindings {
+				for key, value := range pod.Spec.NodeSelector {
+					Expect(node.Labels).ToNot(HaveKeyWithValue(key, value))
```

Comment:
Shouldn't this be the following? Why wouldn't we want the node labels to have the nodeSelectorLabels?
```suggestion
					Expect(node.Labels).To(HaveKeyWithValue(key, value))
```